app/src/docs/Assets/Networks/Interfaces.tsx

```diff
@@ -0,0 +1,115 @@
+/* @license Copyright 2023 @polkadot-cloud/library authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import { SimpleEditor } from "@docs/SimpleEditor";
+
+export const NetworkInterface = () => {
+  const code = `export interface NetworkInformation {
+    name: string;
+    network_type: string;
+    chain: Chain;
+    chainspec: Chainspec;
+    goals: any[];
```

Comment:
Is this just an array of strings?

```suggestion
    goals: string[];
```